index.html

```diff
@@ -937,20 +937,32 @@ <h2>
         </p>
       </section>
     </section>
-    <section class="informative">
+    <section>
       <h2>
         Privacy and Security Considerations
       </h2>
-      <section>
-        <h3>
-          Access to aspects of a user's local computing environment
-        </h3>
-        <p>
-          The screen orientation type and angle of the device can be accessed
-          with the API specified in this document, and can be a potential
-          fingerprinting vector.
-        </p>
-      </section>
+      <p>
+        A screen's [=current orientation type|type=] and [=current orientation
+        angle|angle=] are a potential fingerprinting vector. To resist
+        fingerprinting (e.g., in private browsing), user agents MAY:
+      </p>
+      <ol>
+        <li>Restrict the value return by the {{ScreenOrientation/type}}
+        attribute to {{OrientationType/"portrait-primary"}} or
+        {{OrientationType/"landscape-secondary"}} to match the screen's aspect
+        ratio.
+        </li>
+        <li>Always return `0` for the value of the {{ScreenOrientation/angle}}
+        attribute.
+        </li>
```

Comment:
Shouldn't this be 0 or 90 to match the thing above? Or maybe angle is more complicated in which case this might warrant some further documentation here and in the angle domintro as this would mean you cannot rely on it changing.

Reply:
Yeah, I was chewing on this also... I like the idea of returning `90` for landscape-primary if it's obvious (potentially derived by UA string) that the device has a natural orientation of portrait. Otherwise, it could be used to detect if the user is in private browsing mode.